docs/quickstart.md

```diff
@@ -25,44 +25,50 @@ commands need to be run by a user that is in the `docker` group or by the
 `root` user.
 
 Depending on your Engine system configuration, you may be required
-to preface each `docker` command with `sudo`. One way to avoid having to use
-`sudo` with the `docker` commands is to create a Unix group called `docker` and
-add users that will be entering `docker` commands to the 'docker' group.
+to preface each `docker` command with `sudo`. If you want to run without using
+`sudo` with the `docker` commands, then create a Unix group called `docker` and
+add the user to the 'docker' group.
 
 For more information about installing Docker Engine or `sudo` configuration, refer to
 the [installation](installation/index.md) instructions for your operating system.
 
 
 ## Download a pre-built image
+To pull an `ubuntu` image, run:
 
     # Download an ubuntu image
     $ docker pull ubuntu
 
-This will find the `ubuntu` image by name on
-[*Docker Hub*](userguide/containers/dockerrepos.md#searching-for-images)
+This downloads the `ubuntu` image by name from [Docker Hub](https://hub.docker.com) to a local
+image cache. To search for an image, run `docker search`. For more information, go to:
+[*Searching images](userguide/containers/dockerrepos.md#searching-for-images)
```

Comment:
stray `*` here; I don't think we need this to be cursive, so you can remove that